syntax = "proto3";

package helloworld;

service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {}
  rpc SayGoodBye (GoodByeRequest) returns (GoodByeReply) {}

}

message HelloRequest {
  string name = 1;
  string lastName = 2;
}

message HelloReply {
  string message = 1;
  string full_name = 2;
}

message GoodByeRequest {

}

message GoodByeReply {
  string message = 1;
}